sample_input_files/data_dictionary.md

```diff
@@ -35,8 +35,12 @@ This is essentially a weighted edge list.
 * Second column is the destination location label
 * Third column is the number of individuals undertaking that journey as reported in wu01uk
 
-## sample_20200327_comix_social_contacts.sampleCSV
-This is a sample square matrix of mixing - each column and row header is an age category. For a simplified version, look at simplified_age_infection_matrix.csv.
+## sample_20200327_comix_social_contacts.csv
+This is a sample square matrix of mixing - each column and row header is an age category. For a simplified version, look at simplified_age_infection_matrix_from_comix.csv.  It is from https://cmmid.github.io/topics/covid19/comix-impact-of-physical-distance-measures-on-transmission-in-the-UK.html, and includes all imputed contacts (both face-to-face conversation and physical contact).
```

Comment:
Would it be better described *not* as a sample but as the actual raw data?

Reply:
I more meant that this is a particular contact matrix - there are several others, even within the consortium.  I'll change 'sample' to 'possible'